Libplanet.Explorer/GraphTypes/IValueType.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Bencodex;
+using GraphQL.Language.AST;
+using GraphQL.Types;
+
+namespace Libplanet.Explorer.GraphTypes
+{
+    public class IValueType : StringGraphType
```

Comment:
I think this type can inherit `ScalarGraphType`.

Reply:
afik `StringGraphType` has been inherited from `ScalarGraphType`. did you mean that inheriting `ScalarGraphType` is better than `StringGraphType`? (Actually, I referenced `PublicKeyType` 😅 )

Reply:
Yes, it was because the document example does and I don't know what parts of the `StringGraphType` are needed 🤔

Reply:
On my understanding, `StringGraphType` can be natural because encoded value will be expressed as (hex) string value. (but I don't have strong opinion, I copied that from `PublicKeyType` in our project 😕)